css/style.css

```diff
@@ -20,22 +22,22 @@ main {
 .screen {
     height: 800px;
     border: 1px solid #000000;
+    background-color: #0e8ecc;
+    box-shadow: inset 0 0 10px rgba(0,0,0,0.7);
 }
 
 #splash_screen {
     display: flex;
     justify-content: center;
     align-items: center;
-}
-
-#start {
-    padding: 40px 80px;
-    font-size: 2.5rem;
+    filter: drop-shadow(5px 5px 5px rgba(0,0,0,0.2));
 }
 
 .instructions {
     border: 1px solid #000000;
     margin-top: 30px;
+    background-color: #3FA142;
+    box-shadow: inset 0 0 10px rgba(0,0,0,0.7);
```

Comment:
can we have some space under the instructions? just like 20px to make it easier to read